pyvenv/ufswm.txt

```diff
@@ -3,6 +3,5 @@ Cython >= 0.29.21
 numpy >= 1.19.4
 pandas >= 1.1.5
 python-dateutil >= 2.8.1
-h5netcdf >= 0.8.1
```

Comment:
I guess you are leaving it up to netcdf4 to figure out its dependencies.

Reply:
yes. That one is tricky because of incompatibilities on which version of netcdf-python is used.  Or atleast that is what I found.